pennylane/interfaces/__init__.py

```diff
@@ -16,20 +16,23 @@
 
 PennyLane provides support for various classical
 machine learning interfaces, including Autograd/NumPy, PyTorch,
-and TensorFlow.
+and TensorFlow. The interfaces have access to gradients of a QNode, and
+can therefore integrate quantum computations into the machine learning-library
+pipeline for machine learning and optimization.
```

Comment:
this sentence is a bit awkward. suggest further polish, e.g.,
"can therefore integrate quantum computations into a larger machine learning or optimization pipeline."

Reply:
Much better 👍